CMakeLists.txt

```diff
@@ -20,7 +20,7 @@
 cmake_minimum_required(VERSION 3.17)
 cmake_policy(SET CMP0096 NEW) # policy to preserve the leading zeros in PROJECT_VERSION_{MAJOR,MINOR,PATCH,TWEAK}
 cmake_policy(SET CMP0065 OLD) # default export policy, required for self-dlopen
-project(nifi-minifi-cpp VERSION 0.12.0)
+project(nifi-minifi-cpp VERSION 0.13.0)
```

Comment:
There is another version number in libminifi/CMakeLists.txt

Reply:
thanks -- fixed in 85fb3f6dbccb8588774173d2b80c832c07e27054